user/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+from .views import UserIdentityCheck
+from .views import UserSignUp , SigninView
+urlpatterns = [
+    path('/idCheck', UserIdentityCheck.as_view()),
+    path('/signUp', UserSignUp.as_view())
+    path('/signin', SigninView.as_view())
+     ]
```

Comment:
1. python module import 순서는 다음과 같아요! 기억해주세요!

    ```python
    # 외부 module
    
    # build-in module
    
    # 내가 만들 module
    
    ```
2. import 문과 urlpatterns 사이에 한 줄 띄워주세요. 그리고 line6 마지막에 `,`가 빠진 것 같은데 실행되나요?
    ```suggestion
    from django.urls import path
    
    from .views import UserIdentityCheck
    from .views import UserSignUp , SigninView
    
    urlpatterns = [
        path('/idCheck', UserIdentityCheck.as_view()),
        path('/signUp', UserSignUp.as_view()),
        path('/signIn', SigninView.as_view())
    ]
    ```